<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1577</width>
    <height>942</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>460</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wartość Zadana i Regulowana</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>810</y>
      <width>281</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wartość Uchybu Regulacji</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>810</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wartości Regulatora PID</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>-30</y>
      <width>1301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <italic>false</italic>
      <underline>false</underline>
     </font>
    </property>
    <property name="text">
     <string>Symulator Układu Automatycznej Regulacji</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>220</y>
      <width>171</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>330</y>
      <width>171</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>460</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Reset ID</string>
    </property>
   </widget>
   <widget class="QWidget" name="wykresWarZad" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>110</y>
      <width>931</width>
      <height>351</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="WykUchyb" native="true">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>540</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="WykPID" native="true">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>540</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="UstawieniaGeneratora">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>630</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Generator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UstawieniaRegulatora">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>770</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Regulator</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UstawieniaObiektuARX">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>700</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Obiekt ARX</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>580</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ustawienia</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>840</y>
      <width>167</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Interwał Czasowy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Interwal">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>870</y>
      <width>167</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="WykSter" native="true">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>540</y>
      <width>311</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>810</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Wartość Sterująca</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1577</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
